stable/jenkins/values.yaml

```diff
@@ -87,6 +87,7 @@ Agent:
   # - type: Secret
   #   secretName: mysecret
   #   mountPath: /var/myapp/mysecret
+  NodeSelector: {}
```

Comment:
Yeah I don't mind that idea at all. Let me get back to you on that. Thanks @kfox1111